op-program/client/l2/tmdb.go

```diff
@@ -81,40 +81,12 @@ func (o OracleIAVLKeyValueStore) NewBatch() dbm.Batch {
 	}
 }
 
-// Set satisfies db.DB
-func (o OracleIAVLKeyValueStore) Set(key, val []byte) error {
```

Comment:
are none of these needed anymore?

Reply:
Yeah these are no longer exposed on the iavl.DB interface used to back the v1 version of the IAVL